File: server_ejb/src/main/java/com/example/dto/CompteCourantDto.java
package com.example.dto;

import java.time.LocalDate;

import lombok.Data;

@Data
public class CompteCourantDto {
    
    private Integer idCompte;
    private String nom;
    private Integer idClient;
    private Double capital;
    private LocalDate dateOuverture  = LocalDate.now();
    private Double decouvertAutorise = 0.0;
}

File: pret/ExternalApi/DTO/CompteCourantDto.cs
using System;

namespace Pret.ExternalApi.DTO
{
    public class CompteCourantDto
    {
        public int IdCompte { get; set; }
        public  string Nom { get; set; }
        public double Solde { get; set; }
        public double DecouvertAutorise { get; set; }
    }

    
}
